test/test_images/printport/README.md

```diff
@@ -0,0 +1,21 @@
+# PrintPort Test Image
```

Comment:
This is okay, but I'd sort of prefer a generic container which printed all its env vars, and which could be used in multiple tests.  This is fine for now, though.

Reply:
I agree, but I am going to open a new issue for this.